runtime/queries/query/injections.scm

```diff
@@ -0,0 +1,6 @@
+((predicate 
+  name: (identifier) @_name
+  parameters: (parameters (string) @regex))
+ (#match? @_name "^#?(not-)?(match|vim-match|lua-match)$"))
+
+(comment) @comment
```

Comment:
I think longer term we might want the injections for the comment parser to be kept together in a single package. Since `@comment` is standardized (unlike the node names), we could probably use that as an injection point somehow.

Reply:
Yes, we might want to have some "default" queries, although that might a) be harder to reason about and b) make sharing queries harder.

Personally, I don't like that injection at all -- to simplify things, I could just drop the injection query from this PR altogether, which would avoid any discussion of forking/deviating?

Reply:
(I have removed the comment injections from the other queries I've upstreamed...)

Reply:
Ok, remove it for now.

Reply:
(but these are precisely the general discussions we need to have while starting to add these as official Neovim runtime files, so +1)

Reply:
The `comment` parser is divisive, I personally don't mind it, but it is a good example of something to help identify requirements (a general parser that can be injected in almost any language) so I woudln't be against bundling this too.